src/olympia/versions/models.py

```diff
@@ -170,7 +170,7 @@ class Version(OnChangeMixin, ModelBase):
     deleted = models.BooleanField(default=False)
 
     source = models.FileField(
-        upload_to=source_upload_path, null=True, blank=True)
+        upload_to=source_upload_path, null=True, blank=True, max_length=255)
```

Comment:
mm a migration on the `versions` table to fix a failing test case? What was the max before?

Reply:
also, is the added constraint python/django only?

Reply:
The migration is because I changed the model field (the field change was to fix the failing test case).  https://docs.djangoproject.com/en/2.2/ref/models/fields/#filefield the default is 100.

Reply:
> also, is the added constraint python/django only?

That's what the migration is for - to update the database too.

Reply:
Do we really need that ? Migrations on the `versions` table are annoying for ops...

Reply:
Well, alternatives welcome.  Everywhere else we handle file path/names we use CharField with max_length=255 (at least the places I could find) but here we've used `FileField` and not overridden the max_length so it's 100.  If we want to change that it needs a DB migration.

It could theoretically happen with a user upload too if they attempted to upload a file with a long enough name.

Reply:
I would hope users would simply get a `ValidationError` out of it... but maybe not since the directory also matters? Anyway, I guess that's fine. We'll need to warn ops ahead of time so that the migration can finish, and be aware the push might take a few minutes longer.

Reply:
> That's what the migration is for - to update the database too.

Not all migrations perform operations on the db.

I was asking because, as @diox said, migrations on tables like `versions` aren't simple.